v1/proto/service/gribi.proto

```diff
@@ -113,6 +113,13 @@ message ModifyRequest {
   // conidered master, in order to allow for a client to reconnect
   // without an external election having taken place.
   //
+  // Unpon discovering a failover, the server should stop processing pending
+  // AFTOperations that were sent by the previous master. For cases that the
+  // processing can not be cancelled, it is also ok for the server to continue
+  // finishing the processing and sending the corresponding responses to the
+  // previous master. Clients are expected to make effort for a smooth failover
```

Comment:
To be more precise here, we're saying that "if the acquiring-master client needs the server to be consistent with its own state, it must handle this via a reconciliation process. Such a process can be implemented by acquiring the current state of the server via Get, resetting it via Flush, and using Modify to bring the state in-line with the expectations of the client."

Reply:
I'm removing this part wording. Reading it again, I feel like a spec doc or the motivation.md probably is a better place for it.